src/crimson/osd/osd_operations/client_request.cc

```diff
@@ -402,12 +401,6 @@ ClientRequest::process_op(
   DEBUGDPP("{}.{}: past scrub blocker, getting obc",
 	   *pg, *this, this_instance_id);
 
-  auto obc_manager = pg->obc_loader.get_obc_manager(m->get_hobj());
-
-  // initiate load_and_lock in order, but wait concurrently
-  ihref.enter_stage_sync(
-      client_pp(*pg).lock_obc, *this);
```

Comment:
Ack, this becomes unnecessary because `process` is changed from per-pg to per-object exclusive ordered.